src/uwtools/drivers/driver.py

```diff
@@ -163,6 +172,7 @@ def _resources(self) -> Dict[str, Any]:
             "account": platform["account"],
             "rundir": self._rundir,
             "scheduler": platform["scheduler"],
+            "stdout": "%s.out" % self._runscript_path.name,  # config may override
```

Comment:
Provide a default output filename based on the runscript name, to be used in case the user does not specify anything. This will almost certainly be better than whatever name is chosen by the batch scheduler.

Reply:
I was just noticing this in my example yesterday!